File: sleuth-example/src/main/java/me/brunosantana/controller/SleuthController.java
package me.brunosantana.controller;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class SleuthController {

    private Logger logger = LoggerFactory.getLogger(this.getClass());

    @GetMapping("/")
    public String helloSleuth() {
        logger.info("Hello Sleuth");
        return "success";
    }

}

File: spring-sleuth/src/main/kotlin/com/example/springsleuth/controller/SleuthController.kt
package com.example.springsleuth.controller

import org.slf4j.LoggerFactory
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class SleuthController {

    private val logger = LoggerFactory.getLogger(javaClass);

    @GetMapping("/")
    public fun helloSleuth(): String {
        logger.info("Hello Sleuth");
        return "success!";
    }

}

